addons/pos_self_order/static/src/kiosk/pages/landing_page/landing_page.js

```diff
@@ -0,0 +1,52 @@
+/** @odoo-module */
+
+import { Component, onWillStart } from "@odoo/owl";
+import { useselfOrder } from "@pos_self_order/kiosk/self_order_kiosk_service";
+import { useService } from "@web/core/utils/hooks";
+import { KioskTemplate } from "@pos_self_order/kiosk/template/kiosk_template";
+import { LanguagePopup } from "@pos_self_order/kiosk/components/language_popup/language_popup";
+import { _t } from "@web/core/l10n/translation";
+import { Order } from "@pos_self_order/common/models/order";
+
+export class LandingPage extends Component {
+    static template = "pos_self_order.LandingPage";
+    static components = { KioskTemplate };
+
+    setup() {
+        this.selfOrder = useselfOrder();
+        this.router = useService("router");
+        this.dialog = useService("dialog");
+
+        onWillStart(() => {
+            this.selfOrder.currentOrder = new Order({});
+        });
+    }
+
+    start() {
+        if (this.selfOrder.kiosk_takeaway) {
+            this.router.navigate("location");
+        } else {
+            this.router.navigate("product_list");
+        }
+    }
+
+    openLanguages() {
+        this.dialog.add(LanguagePopup, {
+            title: _t("Cancel order"),
+            body: _t(
+                "Are you sure you want to cancel the order? All the products in the cart will be removed."
+            ),
+            confirm: () => {
+                this.router.navigate("default");
+            },
```

Comment:
I think the LanguagePopup does not use neither the body nor the confirm props. This could certainly be removed